src/components/Article.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ input }) => (
```

Comment:
```suggestion
const Article = ({ title, date, text }) => (
```
If you have passed the whole `article` object as I've mentioned above, you could use destructuring right away